gatsby-theme-portfolio-minimal/src/sections/About/index.tsx

```diff
@@ -8,20 +8,25 @@ import * as classes from './style.module.css';
 
 export function AboutSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
-    const data = response.allAboutMarkdown.sections[0];
+    const allFiles = response.allFile.aboutFiles;
+    const fileNameNeedle = props.fileName ? props.fileName : 'about';
+    const result = allFiles.find((file) => {
+        return file.name == fileNameNeedle;
+    });
+    const section = result ? result.section[0] : allFiles[0].section[0];
```

Comment:
Would be great if we could abstract this away + add null checking and error messages for unknown sections. For example, for the Legal sections, we did: 

```js
const data = getSectionBySectionId(response, props.sectionId);
```

and handled the data part entirely in `Legal/data.tsx`.